### PR TITLE
Update approval-admin.md

### PR DESCRIPTION
**Pull request by ConnorR777**
Added Approvals - Parent App section as the feature is releasing. This will help Admins understand the integration between the two apps.

### CODE REVIEW

File: Teams/approval-admin.md [hunk 1]
```diff
@@ -229,6 +229,22 @@ You can use the Teams admin center to control which third-party e-signature prov
 
 E-signature approvals created from the Approvals app are stored in the selected provider's cloud environment. To export data about e-signatures, you'll need to go to the provider's site. For more information about storage, export, and retention of e-signature agreements, see the provider's documentation.
 
+## Approvals - Parent App integration
+
+For teachers and admins using Teams for education, they can use [Parent App](/https://aka.ms/parentapp) to update the contact information for class students’ parents. 
+
+Admins can use the approvals app to view and approve/ reject the request raised by schoolteachers to update the parent contact information accordingly. Once admin approves this request, the new contact details will start reflecting inside the parent app and allow teachers to communicate with the parent.
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:236]:
```suggestion
Teachers and admins using Teams for education can use the [Parent App](/https://aka.ms/parentapp) to update the contact information for class students’ parents. 

Admins can use the approvals app to view and approve or reject the request raised by schoolteachers to update the parent contact information accordingly. Once the admin approves this request, the new contact details will reflect in the parent app and allow teachers to communicate with the parent.
```

File: Teams/approval-admin.md [hunk 2]
```diff
@@ -229,6 +229,22 @@ You can use the Teams admin center to control which third-party e-signature prov
...
+
+### Frequently Asked Questions:
+
+#### Question: How does the teacher track the status of the approval request?
+
+The teacher can navigate to the Approvals app within Teams -> Select “sent” tab -> Select Filter and search by title “Approval request to update Parent/Guardian details”. This allows a view of all sent requests for parent information update.
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:242]:
```suggestion
The teacher should navigate to the Approvals app within Teams. Select **Sent** > **Filter** and search for the title **Approval request to update Parent/Guardian details**. This allows a view of all sent requests for parent information updates.
```

File: Teams/approval-admin.md [hunk 3]
```diff
@@ -229,6 +229,22 @@ You can use the Teams admin center to control which third-party e-signature prov
...
+
+#### Question: How long does it take for new contact details to reflect inside the Parent App?
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:244]:
```suggestion
#### Question: How long does it take for new contact details to reflect in the Parent App?
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ConnorR777, kindly commit the suggested changes. Many thanks.

**Comment by ConnorR777** [comment]
@JohanFreelancer9 

> Copy review - @ConnorR777, kindly commit the suggested changes. Many thanks.

Done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ConnorR777** [comment]
@JohanFreelancer9 any thing more needed to get the changes merged? Thanks!

**Pull request merged**